compiler/modules/types/list.bal

```diff
@@ -332,6 +285,43 @@ function listInhabited(Context cx, SemType[] m, int repeatLastCount, SemType res
     }
 }
 
+function shallowCopyListAtomicType(ListAtomicTypeMembers members) returns ListAtomicTypeMembers {
+    return { initial: shallowCopyTypes(members.initial), repeatLastCount: members.repeatLastCount };
+}
+
+function listMemberAt(ListAtomicTypeMembers members, int i) returns SemType {
+    return members.initial[int:min(i, members.initial.length() - 1)];
+}
+
+function isSameListAcomicTypeStructure(ListAtomicTypeMembers v1, ListAtomicTypeMembers v2) returns boolean {
+    return v1.repeatLastCount == v2.repeatLastCount && v1.initial.length() == v2.initial.length();
+}
+
+function listDecompressMembersForSet(ListAtomicTypeMembers members, int setIndex, SemType withNewMember) {
```

Comment:
This is just performing a set-member operation of `ListAtomicTypeMembers`; `listMemberAt` is performing a get-member. The naming of the two functions should reflect this.

Reply:
1. This doesn't set when the `setIndex` is below the length of `initial` list. Can we say this is the set-member?

2. How about `listMemberGet`?

Reply:
> This doesn't set when the setIndex is below the length of initial list.

Can we change things so it does?

Reply:
I think we need to pass the `rest` type to this as we sometime need to fill with `rest` type too.

Reply:
Will update the PR with that change